tests/test_tui.py

```
@@ -0,0 +1,110 @@
+
+    def test_diff_line(self):
+        assert _escape_markup("+[new] line") == "+\\[new] line"
+
+
+class TestLogoBanner:
+    def test_banner_has_content(self):
+        assert len(LOGO_BANNER) > 0
+
+    def test_banner_is_3_lines(self):
+        lines = LOGO_BANNER.split("\n")
+        non_empty = [l for l in lines if l.strip()]
```

Ruff (0.15.13)

[error] Ambiguous variable name: `l`

(E741)